docker/test

```diff
@@ -75,29 +74,51 @@ then
   exit 1
 fi
 
+if [ "$COMPOSE_ARGS" == "-f $DIR/docker-compose-postgres-cs9.yml" ]; then
+  INIT_CONTAINER="1"
+fi
+
 # PROJ_NAME should be set in a jenkins environment. It allows multiple
 #  instances of the compose to run without clobbering each other.
 cd $DIR
-docker-compose $PROJ_NAME stop
-docker-compose $PROJ_NAME rm -f
 
-if [ "$USE_CACHE" != "1" ]; then
-  docker-compose $COMPOSE_ARGS pull
-fi
 
 SERVICE_PORTS_ARG=""
 if [ "$EXPOSE_PORTS" == "1" ]; then
   SERVICE_PORTS_ARG="--service-ports"
 fi
 
-if [ "$DETACHED" == "1" ]; then
+if [ "$INIT_CONTAINER" == "1" ]; then # For CS9
+  podman-compose $PROJ_NAME stop
+  podman-compose $PROJ_NAME rm -f
+
+  if [ "$USE_CACHE" != "1" ]; then
+    podman-compose $COMPOSE_ARGS pull
+  fi
+  podman-compose $PROJ_NAME $COMPOSE_ARGS up -d
+  sleep # Waiting to start systemd
+  podman-compose $PROJ_NAME $COMPOSE_ARGS exec candlepin $TEST_CMD
```

Comment:
if DETACHED is set, then the `exec` needs to run with `-d`

Reply:
I just tried this, and it doesn't seem to work. podman still runs the container in the foreground

Reply:
Let's use podman direct commands (e.g. `podman pod create`, `podman start/exec`)

Reply:
done